test/pulpcore_plugin_integration_test.rb

```diff
@@ -40,10 +46,11 @@ def test_features
   def test_features_with_optional_params
     Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulpcore.yml').returns(
       enabled: true,
-      pulp_url: 'http://pulpcore.example.com/foo',
+      pulp_url: 'http://pulpcore.example.com/pulpcore',
```

Comment:
@ekohl I kept this as /pulpcore for now to differentiate from /rhsm

that said, if it's necessary here to show that this can be any arbitrary url, I can change these as well

Reply:
I still think this is an irrelevant change. Those only make reviewing a patch harder. There is no reason that it must be `/pulpcore`. Someone may deploy it to `/foo` if they want to. It's not like we default to `/pulpcore` anywhere either so this is changing it from one theoretical example to another.

Reply:
sure, it's really just a readability change. when introducing more services it's easier for me to think about when there is some pattern rather than arbitrarily assigning `foo`, `bar`, `baz`

Reply:
To me the hostname should be sufficient. I very explicitly chose this URL and I think changing it is bad. I tried to say "Pulp can run on any arbitrary hostname with any arbitrary path". You are changing that.

Reply:
OK, it's reverted back to pulpcore.example.com/foo